app/containers/message/Message.stories.tsx

```diff
@@ -80,13 +81,37 @@ export const Message = (props: any) => (
 	/>
 );
 
+// The large font components are not perfect because the text's font scale increases only with the device's font size setting.
+export const MessageLargeFont = (props: any) => (
+	<MockResponsiveFontScaleProvider fontScale={1.4}>
```

Comment:
This is just a mock and there's a prop with 1.4 and a default value on your provider with 1.4
Either one should be used, but not both.